<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Appointment</class>
 <widget class="QWidget" name="Appointment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/images/icon-512.png</normaloff>:/images/icon-512.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">border: none;
background-color: rgb(31, 31, 39);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="date0">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(64, 64, 80, 100);padding: 3px;border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>07.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date1">
        <property name="text">
         <string>08.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date2">
        <property name="text">
         <string>09.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date3">
        <property name="text">
         <string>10.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date4">
        <property name="text">
         <string>11.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date5">
        <property name="text">
         <string>12.05</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="date6">
        <property name="text">
         <string>13.05</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="appointments_scroll">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>533</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
